### PR TITLE
Add ARM64 CI w/ Travis

### PR DESCRIPTION
**Pull request by jadr2ddude**
Since travis supports ARM64, why not test ARM64 in CI?

### CODE REVIEW

**Comment by aykevl** [comment]
If you hadn't seen already, these might be of interest: https://github.com/tinygo-org/tinygo/pull/261 and https://github.com/tinygo-org/tinygo/pull/210.

**Comment by jadr2ddude** [comment]
Alright, this should be ready.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - llvm-toolchain-xenial-8
+      - sourceline: 'deb https://deb.nodesource.com/node_10.x bionic main'
+        key_url: 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key'
+    packages:
+      - clang-8
+      - llvm-8-dev
+      - libclang-8-dev
+      - lld-8
+      - gcc-arm-linux-gnueabihf
+      - binutils-arm-none-eabi
```
Comment (aykevl) [inline .travis.yml]: I believe binutils are not necessary. Have you tried running without?

**Comment by jadr2ddude** [comment]
Trying without binutils now.

File: .travis.yml (hunk 1 above)
Comment (jadr2ddude) [reply .travis.yml]: Trying without binutils.

**Comment by jadr2ddude** [comment]
Alright, feel free to merge now if you think this is good.

**Comment by deadprogram** [comment]
We're not using TravisCI anymore that I know of, we switched over to CircleCI some time ago. Are you proposing to use it only for this architecture? As of right now, we have both CircleCI and also Azure, I'd personally prefer not to have to use/administer/maintain 3 different CI systems...

**Comment by jadr2ddude** [comment]
Yes. This is only to use it for this architecture.

>I'd personally prefer not to have to use/administer/maintain 3 different CI systems...
Neither would I, but I think it is worth it to run CI on a different architecture.

**Comment by aykevl** [comment]
I'm not too happy with 3 CI providers either, but it does allow us to add tests for ARM. It has been requested for CircleCI, but hasn't yet been implemented: https://ideas.circleci.com/ideas/CCI-I-900 (please vote there!).

**Comment by jadr2ddude** [comment]
Have we decided on whether we would like to merge this?

**Comment by deadprogram** [comment]
I propose that we just wait until this architecture is available in CircleCI to keep the current build complexity to only two providers.

**Comment by jadr2ddude** [comment]
The general consensus appears to be that this does not make sense.

**Pull request closed without merging**